webview/src/shared/components/sectionContainer/styles.module.scss

```diff
@@ -1,5 +1,5 @@
 @import '../../variables.scss';
-@import '../../style.scss';
+@import '../../styles.scss';
```

Comment:
Same here, I don't think we need to import this.

Reply:
Good catch!